src/tests/Home.test.js

```diff
@@ -0,0 +1,17 @@
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import renderer from 'react-test-renderer';
+import Home from '../components/Home';
+
+describe('Home', () => {
+  test('should render the home component', () => {
+    const tree = renderer.create(<Home />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
```

Comment:
- [ ] Great job so far Team :muscle: , you did an amazing job writing all the tests for your components. However, this particular snapshot test failed when I run the tests suite. So kindly fix this so all your tests can pass successfully :+1: 
- I have attached a screenshot reference below :100: 
![image](https://user-images.githubusercontent.com/102699958/229075970-df2ea17c-79a4-4c47-9efd-3686260ac3a6.png)

Reply:
> * [ ]  Great job so far Team 💪 , you did an amazing job writing all the tests for your components. However, this particular snapshot test failed when I run the tests suite. So kindly fix this so all your tests can pass successfully 👍
> * I have attached a screenshot reference below 💯
>   ![image](https://user-images.githubusercontent.com/102699958/229075970-df2ea17c-79a4-4c47-9efd-3686260ac3a6.png)

Thanks @NahnahAJ for your feedback, I have now fixed the test and all passing.